tests/python/gaia-ui-tests/gaiatest/apps/homescreen/regions/search_panel.py

```diff
@@ -93,10 +94,7 @@ def app_results(self):
 
     @property
     def link_results(self):
-        # A link result just opens a page in a frame
-        return [self.Result(marionette=self.marionette, element=result)
-                for result in self.marionette.find_elements(*self._search_results_locator)
-                    if not self._is_result_a_webapp(result)]
+        return self.marionette.find_elements(*self._link_results_locator)
```

Comment:
why did we removed the `Result` object in this and we return only a list of webelements